build.sh

```diff
@@ -3,23 +3,23 @@
 project_name=placeholder
 subproject_name=main
 echo Hello, this script will only copy folders in the correct format for our hosted website
-echo Project used on / - $project_name
-echo Project used on /preview - $subproject_name
+echo Project used on / - \"$project_name\"
+echo Project used on /preview - \"$subproject_name\"
 
-build_project() {
-    echo Build main project $project_name
-    npm --prefix src/$project_name/ --cwd src/$project_name/ install src/$project_name/
-    npm --prefix src/$project_name/--cwd src/$project_name/ run build
+build_mainproject() {
+    echo Build main project \"$project_name\"
+    npm --prefix src/$project_name/  install src/$project_name/
+    npm --prefix src/$project_name/ run build
 }
 
 build_subproject() {
-    echo Build subproject $subproject_name
-    npm --prefix src/$project_name/ --cwd src/$subproject_name/ install src/$subproject_name/
-    npm --prefix src/$project_name/ --cwd src/$subproject_name/ run build
+    echo Build sub project \"$subproject_name\"
```

Comment:
I don't know if this is a good point, but I wanted to ask if space in "sub project" was intentional and would not be problematic.